rdkit-postgresql/cmakelists.txt.patch

```diff
@@ -1,18 +1,68 @@
 diff --git Code/PgSQL/rdkit/CMakeLists.txt Code/PgSQL/rdkit/CMakeLists.txt
-index 2ef36c7..b46a119 100644
+index fb062165..d565e32d 100644
 --- Code/PgSQL/rdkit/CMakeLists.txt
 +++ Code/PgSQL/rdkit/CMakeLists.txt
-@@ -1,5 +1,13 @@
+@@ -1,5 +1,19 @@
  cmake_minimum_required(VERSION 2.8.8)
  
 +project(RDKitPostgreSQL)
 +enable_testing()
 +
++set(CMAKE_MODULE_PATH
++    ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Modules/")
++
 +find_package(RDKit REQUIRED)
 +include_directories(${RDKit_INCLUDE_DIRS})
 +
-+find_package(PostgreSQL REQUIRED)
++# Boost and Eigen libraries that are required at link time
++find_package(Boost 1.56.0 COMPONENTS serialization thread)

```

Comment:
You should be able to drop thread from the requirements here.

Reply:
I just tried, but cmake produced a warning reading `[...] Target "rdkit" links to target "Threads::Threads" but the target was not found.  Perhaps a find_package() call is missing for an IMPORTED target, or an ALIAS target is missing?` and the build failed with `/home/ric/rdkb/rdkit-postgresql_1525713265598/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.2.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lThreads::Threads`